GetMessages/__tests__/handler.test.ts

```diff
@@ -48,61 +58,306 @@ const aRetrievedPendingMessageWithoutContent: RetrievedMessageWithoutContent = {
   kind: "IRetrievedMessageWithoutContent"
 };
 
+const blobServiceMock = ({
+  getBlobToText: jest.fn()
+} as unknown) as BlobService;
+
+const getMockIterator = values => ({
+  next: jest
+    .fn()
+    .mockImplementationOnce(async () => ({
+      value: values
+    }))
+    .mockImplementationOnce(async () => ({ done: true }))
+});
+
+const getMessageModelMock = messageIterator =>
+  (({
+    getContentFromBlob: () =>
+      TE.of(
+        O.some({
+          subject: "a subject",
+          markdown: "a markdown"
+        } as MessageContent)
+      ),
+    findMessages: jest.fn(() => TE.of(messageIterator))
+  } as unknown) as MessageModel);
+
+const serviceModelMock = ({
+  findLastVersionByModelId: () => TE.of(O.some(aRetrievedService))
+} as unknown) as ServiceModel;
+
 describe("GetMessagesHandler", () => {
```

Comment:
We must add a test case with ResponseErrorQuery

Reply:
Done in https://github.com/pagopa/io-functions-app/commit/11001b447ffe2a0d9beeb733b32fd23ec43971f2